javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/SafeServiceLoader.java

```diff
@@ -40,5 +43,15 @@
     return result;
   }
 
+  /**
+   * Same as {@link #load(Class)}, but also orders the returned implementations by comparing their
+   * {@link Ordered#order()}.
+   */
+  public static <T extends Ordered> List<T> loadOrdered(Class<T> serviceClass) {
+    return load(serviceClass).stream()
+        .sorted(Comparator.comparingInt(Ordered::order))
+        .collect(Collectors.toList());
```

Comment:
Why not just use `List.sort` method? Without any streams.

Reply:
Good point, done